src/Sentry.Unity.Editor.iOS/NativeOptions.cs

```diff
@@ -41,6 +41,11 @@ internal static string Generate(SentryUnityOptions options)
         return nil;
     }}
 
+    {(options.FilterBadGatewayExceptions ? @"options.beforeSend = ^SentryEvent * _Nullable(SentryEvent * _Nonnull event) {
+        if ([event.request.url containsString:@""operate-sdk-telemetry.unity3d.com""]) return nil;
```

Comment:
any chance `event.request` or `.url` may be null? Would that still work without crashing/throwing?

Reply:
In case they are `nil` the if would fail. But that's oke. We don't want to filter those anyway.